### PR TITLE
Add login feature

### PR DESCRIPTION
**Pull request by jelneo**

### CODE REVIEW

File: src/main/java/seedu/address/logic/Password.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+//@@author jelneo
+/**
+ * Represents the password of a user account
+ */
+public class Password {
+    public static final int PASSWORD_LENGTH = 6;
```
Comment (khooroko) [inline src/main/java/seedu/address/logic/Password.java]: Perhaps you want to rename this? This constant name seems to suggest that the password length has to be exactly 6 characters long

File: src/test/java/seedu/address/logic/PasswordTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        // password meets length requirement
+        assertTrue(Password.isValidPasswordLength("abcde12345"));
+        assertTrue(Password.isValidPasswordLength("ABCE1234!@#"));
+    }
+
+    @Test
+    public void isValidPassword() {
+        // blank password
+        assertFalse(Password.isValidPasswordLength(""));
+        assertFalse(Password.isValidPasswordLength(" "));
```
Comment (khooroko) [inline src/test/java/seedu/address/logic/PasswordTest.java]: Should these two be .isValidPassword instead?

**Review by khooroko — APPROVED** [review]
Minor issues in naming but aside from that really great job :)

File: src/main/java/seedu/address/logic/Username.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+//@@author jelneo
+/**
+ * Represents the username of a user account
+ */
+public class Username {
+
+    public static final int USERNAME_LENGTH = 6;
```
Comment (khooroko) [inline src/main/java/seedu/address/logic/Username.java]: Same as above

File: src/main/java/seedu/address/logic/Password.java (hunk 1 above)
Comment (jelneo) [reply src/main/java/seedu/address/logic/Password.java]: corrected

File: src/main/java/seedu/address/logic/Username.java (hunk 3 above)
Comment (jelneo) [reply src/main/java/seedu/address/logic/Username.java]: corrected

File: src/test/java/seedu/address/logic/PasswordTest.java (hunk 2 above)
Comment (jelneo) [reply src/test/java/seedu/address/logic/PasswordTest.java]: renamed

**Review by lawwman — APPROVED** [review]
Approved.
Good header comments used.

**Pull request merged**